dotcom-rendering/src/components/SetABTests.importable.tsx

```diff
@@ -54,20 +84,13 @@ export const SetABTests = ({
 	useEffect(() => {
 		if (!ophan) return;
 
-		const mvtId = Number(
-			(isDev &&
-				getCookie({
-					name: 'GU_mvt_id_local',
-					shouldMemoize: true,
-					// eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing -- it would break the logic
-				})) || // Simplify localhost testing by creating a different mvt id
-				getCookie({ name: 'GU_mvt_id', shouldMemoize: true }),
-		);
-		if (!mvtId) {
-			// 0 is default and falsy here
-			console.log(
+		const mvtId = isDev ? getLocalMvtId() ?? getMvtId() : getMvtId();
```

Comment:
I think this would be more confusing, and we have a linting rule against it:

```
Unexpected nullable number value in conditional.
Please handle the nullish/zero/NaN cases explicitly.
```

Using the ternary, we have only three possible states and falsy numbers and strings are avoided entireyl:

1. this is development
  - try to get a valid `number` local mvt id, or `undefined` if it fails
  - if `undefined`, try to get a valid `number` mvt id, or `undefined` if it fails
2. this is prod
  - try to get a valid `number` mvt id, or `undefined` if it fails